plugins/flatpak/tests/build.py

```diff
@@ -80,7 +81,11 @@ def copy_repo(srcdir, destdir):
               'app-missing-runtime/repo')
 
 # app with an update
-copy_repo('app-with-runtime', 'app-update')
+build_flatpak('org.test.Runtime',
```

Comment:
And this should be fine because it just makes the update more complex (not too much though) without compromising the previous tests.
We can (and should) add more tests that do test every single combination (same runtime when updating / new runtime when updating / deleting the runtime right before the update / etc.) but tests are very time taking as they're implemented right now, so I'd prefer to add this now and rework the tests when we have an easier solution to use.

Reply:
As I said before, I don't think this particular thing should stop this PR, but in the future I think it would be better if we add new tests instead of replacing things :-)